### DISCUSSION TITLE
Console Interview

### DISCUSSION THREAD

**Discussion by sjkelleyjr** [kovidgoyal/kitty#4383] [General]
Hey @kovidgoyal!

I run a [newsletter about open source software](https://console.substack.com/) and I like to put an interview with one of the developers in each email.  Would you be interested in answering a few questions about software development and Kitty?

**Comment by kovidgoyal** [comment]
Sure, happy to. You can find my email address at the top of most kitty source files.

**Comment by sjkelleyjr** [comment]
right in front of my face!  I looked all over you github profile and personal website and didn't find anything ha ha! I'll shoot you an email!  Thanks!
